src/errors.ts

```diff
@@ -20,6 +20,8 @@ class ErrorPolykeyClientDestroyed extends ErrorPolykey {}
 
 class ErrorInvalidId extends ErrorPolykey {}
 
+class ErrorInvalidConfigEnvironment extends ErrorPolykey {}
```

Comment:
Not necessary anymore once `--network` is implemented.

Reply:
Replaced with `never()` just in case.

Reply:
This shouldn't be using `never()`. The CLI argument should just use the invalid argument error. There's no way for us to have an invalid "environment".